### PR TITLE
Monaca Power Plugins blog post

### PR DESCRIPTION
**Pull request by KonstantinDinev**
@masahirotanaka when you are available please review my blog post. 
There is a minor issue - the `font-size` of the `p` tags is smaller (`14px`) than the `font-size` of the `li`'s (`17px`). Please let me know if changes has to be done.

### CODE REVIEW

**Comment by andipavllo** [comment]
@KonstantinDinev the build process is broken, add your profile in `config.js` and test if it builds correctly.

**Comment by KonstantinDinev** [comment]
@andipavllo when you are available could you please try to build again?

File: blog/posts/2016-04-08.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+author: konstantin
+date: 2016-04-08
+id: boost-your-cordova-development-with-monaca-power-plugins
+title: "Boost your Cordova Development with Monaca Power Plugins"
+product: monaca
+tags: monaca security, security plugins, security, monaca power, monaca plugins, monaca power plugins, enterprise users, monaca enterprise, enterprise
+category: development
+---
+
+![Monaca Power Plugins](/blog/content/images/2016/Apr/AdobeStockEnc.jpeg)
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: this blog post is not about encryption, the image is misleading, use another image

File: blog/posts/2016-04-08.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+We are glad to announce our new feature plugins to boost your app development experience: Monaca Power Plugins. With our new suite of plugins, we deliver the ultimate protection, providing our customers the ability to secure their work with the latest technologies in terms of security. Let's immediately take a detailed look at the latest updates!
+
+<!-- more -->
+
+### Monaca HTML5 Resource Encryption
+
+This plugin provides the encryption function for HTML5 assets bundled up with the built app in Monaca and the decoding function for application execution time.
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: Reformulate this sentence in a simpler way.

File: blog/posts/2016-04-08.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+The encryption is provided by [AES-256](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) algorithm. Once the HTML5 source code has been converted into binary code, the algorithm is executed, providing the maximum level of security for your resources .
+
+[Read more](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/html5_resource_encryption/)
+
+### Monaca Secure Storage
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: write more about this plugin

File: blog/posts/2016-04-08.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Secure Storage plugin provides an encrypted data storage function. If the application is uninstalled, saved data will also be deleted.
+
+[Read more](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/secure_storage/)
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: don't use external references, download the image and add it to the proper dir

File: blog/posts/2016-04-08.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+author: konstantin
+date: 2016-04-08
+id: boost-your-cordova-development-with-monaca-power-plugins
+title: "Boost your Cordova Development with Monaca Power Plugins"
+product: monaca
+tags: monaca security, security plugins, security, monaca power, monaca plugins, monaca power plugins, enterprise users, monaca enterprise, enterprise
```
Comment (argelius) [inline blog/posts/2016-04-08.markdown]: Please don't add so many tags. 2-3 tags is enough.

File: blog/posts/2016-04-08.markdown [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+[Read more](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/secure_storage/)
+
+### Monaca In-App Updater
+
+This plugin allows to update HTML5 assets contained into the app without any need to rebuild and package everytime. In-App Updater is designed to boost your app development process while providing the comfort of easily update your application with the minimum effort. The only requirement is a Web server to host the updated files which will be accessed by the app.
+
+Here are some easy steps to follow in order to integrate In-App Updater plugin into your app:
+
+1. Enable the plugin. [See Enable Plugin in Monaca.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#enable-in-app-updater)
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: The list font size is higher than the normal content, fix it (use html if needed).

File: blog/posts/2016-04-08.markdown [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+2. Choose the update mode and specify the deploy URL. [See Update Mode.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#update-mode)
+3. Create the update files for intended OS with the proper version. [See Creating Update File and Update File Version.](http://docs.monaca.mobi/cr/en/reference/third_party_phonegap/in-app_updater/#create-update-file)
+4. Create configuration file specifying the version number of the update files. [See Creating Configuration File.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_up    dater/#config-file-in-app-updater)
+5. Place the configuration and update files in the same directory level. [See Placing Update File.]()
+
+
+### Monaca In-App Purchases
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: the content is poor, add more informations

File: blog/posts/2016-04-08.markdown [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+Here are some easy steps to follow in order to integrate In-App Updater plugin into your app:
+
+1. Enable the plugin. [See Enable Plugin in Monaca.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#enable-in-app-updater)
+2. Choose the update mode and specify the deploy URL. [See Update Mode.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#update-mode)
+3. Create the update files for intended OS with the proper version. [See Creating Update File and Update File Version.](http://docs.monaca.mobi/cr/en/reference/third_party_phonegap/in-app_updater/#create-update-file)
+4. Create configuration file specifying the version number of the update files. [See Creating Configuration File.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_up    dater/#config-file-in-app-updater)
+5. Place the configuration and update files in the same directory level. [See Placing Update File.]()
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: missing link

File: blog/posts/2016-04-08.markdown [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+Here are some easy steps to follow in order to integrate In-App Updater plugin into your app:
+
+1. Enable the plugin. [See Enable Plugin in Monaca.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#enable-in-app-updater)
+2. Choose the update mode and specify the deploy URL. [See Update Mode.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#update-mode)
+3. Create the update files for intended OS with the proper version. [See Creating Update File and Update File Version.](http://docs.monaca.mobi/cr/en/reference/third_party_phonegap/in-app_updater/#create-update-file)
+4. Create configuration file specifying the version number of the update files. [See Creating Configuration File.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_up    dater/#config-file-in-app-updater)
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: broken link

File: blog/posts/2016-04-08.markdown [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+### Monaca In-App Updater
+
+This plugin allows to update HTML5 assets contained into the app without any need to rebuild and package everytime. In-App Updater is designed to boost your app development process while providing the comfort of easily update your application with the minimum effort. The only requirement is a Web server to host the updated files which will be accessed by the app.
+
+Here are some easy steps to follow in order to integrate In-App Updater plugin into your app:
+
+1. Enable the plugin. [See Enable Plugin in Monaca.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#enable-in-app-updater)
+2. Choose the update mode and specify the deploy URL. [See Update Mode.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_updater/#update-mode)
+3. Create the update files for intended OS with the proper version. [See Creating Update File and Update File Version.](http://docs.monaca.mobi/cr/en/reference/third_party_phonegap/in-app_updater/#create-update-file)
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: broken link

File: blog/posts/2016-04-08.markdown [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+4. Create configuration file specifying the version number of the update files. [See Creating Configuration File.](http://docs.monaca.mobi/cur/en/reference/third_party_phonegap/in-app_up    dater/#config-file-in-app-updater)
+5. Place the configuration and update files in the same directory level. [See Placing Update File.]()
+
+
+### Monaca In-App Purchases
+
+In-App Purchasing plugin is opening new marketing possibilities for mobile app developers. It greatly improves the profits by offering to the users the opportunity to buy additional content from the applications we create. This functionality could be used in many different applications with different methodologies.
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: `This functionality could be used in many different applications with different methodologies.` doesn't really say too much

File: blog/posts/2016-04-08.markdown [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+
+### How to benefit from our new features
```
Comment (andipavllo) [inline blog/posts/2016-04-08.markdown]: Improve the conclusion

File: blog/authors/konstantin.markdown [hunk 13]
```diff
@@ -0,0 +1,7 @@
+---
+id: konstantin
+name: "Konstantin Dinev"
+gravatar: 31893df4323fac99458ed86784e25a77
+---
+
+Konstantin is a hardware and software engineer. Graduated in the National Military University of Bulgaria.
```
Comment (andipavllo) [inline blog/authors/konstantin.markdown]: Add something more about yourself.

**Comment by andipavllo** [comment]
@KonstantinDinev I made some changes and wrote some comments. Please improve those points.

File: blog/posts/2016-04-08.markdown (hunk 12 above)
Comment (unaso) [inline blog/posts/2016-04-08.markdown]: Please write this part a bit more evoking. The reader should feel excited about trying the new feature and they should see the benefit.

Offering a standalone package is a great news for them, so we can write it in that way...

**Pull request merged**